integration_tests/commands/resp/setup.go

```diff
@@ -127,8 +128,11 @@ func RunTestServer(wg *sync.WaitGroup, opt TestServerOptions) {
 	gec := make(chan error)
 	shardManager := shard.NewShardManager(1, queryWatchChan, cmdWatchChan, gec, logr)
 	workerManager := worker.NewWorkerManager(20000, shardManager)
+	var (
+		cmdWatchSubscriptionChan chan watchmanager.WatchSubscription
+	)
```

Comment:
declare this in the same manner as the queryWatchChan and cmdWatchChan to maintain consistency. Ideally these definitions should be clubbed together.